app/helpers/form_helper.py

```diff
@@ -0,0 +1,115 @@
+from app.helpers.schema_helper import SchemaHelper
+from app.forms.household_composition_form import generate_household_composition_form, deserialise_composition_answers
+from app.forms.household_relationship_form import build_relationship_choices, deserialise_relationship_answers, generate_relationship_form
+from app.forms.questionnaire_form import generate_form
+
+
+def get_form_for_location(block_json, location, answer_store, error_messages, disable_mandatory=False):
```

Comment:
Is there a reason for passing error messages and disabling mandatory fields on get_form?